### ISSUE TITLE
Make announcement clearer

### ISSUE THREAD

**Issue by ErikEJ** [dotnet/SqlClient#2291]
(Unable to comment on the actual announcement, so putting it here)

https://github.com/dotnet/announcements/issues/292

I propose that the Affected Packages table in the announcement should be updated to indicate that 4.x and 5.x series dependencies should also be updated. That is not clear at all from the current version table.

**Comment by David-Engel** [comment]
> I propose that the Affected Packages table in the announcement should be updated to indicate that 4.x and 5.x series dependencies should also be updated. That is not clear at all from the current version table.

I'm not sure what you are asking. Are you talking about the Azure.Identity dependency vulnerability? The announcement is specific to the encryption vulnerability. I think it would get more confusing to add another topic to that announcement.

I did note in my release blog post that we also released 5.1.4 to address the Azure.Identity issue. (Note: We couldn't bump the dependency in 4.x due to major version bumps in the Azure.Identity dependency chain.)
https://techcommunity.microsoft.com/t5/sql-server-blog/released-security-updates-for-microsoft-data-sqlclient-and/ba-p/4024264
